imap_processing/cdf/config/imap_swe_l2_variable_attrs.yaml

```
@@ -0,0 +1,145 @@
+# <=== Coordinates ===>
+esa_step:
+  CATDESC: Energy step id in lookup table
+  FIELDNAM: energy step
+  FILLVAL: -9223370000000000000
+  FORMAT: I2
+  LABLAXIS: Energy Step
+  SCALE_TYP: linear
+  UNITS: " "
+  VALIDMAX: 23
+  VALIDMIN: 0
+  VAR_TYPE: support_data
+
+spin_angle:
+  CATDESC: Spin angle. Angle resolution is 12 degree nominally
+  FIELDNAM: spin angle
+  FILLVAL: -9223370000000000000
+  FORMAT: I2
+  LABLAXIS: Angle
+  SCALE_TYP: linear
+  UNITS: "Degree"
+  VALIDMAX: 29
+  VALIDMIN: 0
+  VAR_TYPE: support_data
+
+cem_id:
+  CATDESC: Data of each CEM detectors
+  FIELDNAM: CEM data
+  FILLVAL: -9223370000000000000
+  FORMAT: E14.7
+  LABLAXIS: Rates
+  SCALE_TYP: linear
+  UNITS: " "
+  VALIDMAX: 0.000015514
+  VALIDMIN: 0.0
+  VAR_NOTES: " "
+  VAR_TYPE: support_data
+
+spin_sector:
+  CATDESC: Spin sector. 30 measurements nominally
+  FIELDNAM: Spin Sector
+  FILLVAL: -9223370000000000000
+  FORMAT: I2
+  LABLAXIS: Spin Sector
+  SCALE_TYP: linear
+  UNITS: " "
+  VALIDMAX: 29
+  VALIDMIN: 0
+  VAR_TYPE: support_data
+
+cycle:
+  CATDESC: Full cycle data takes 4 spins' data
+  FIELDNAM: quater cycle
```

Do you want to call this variable quarter_cycle_id or something similar to that to be in line with cem_id? Or is cycle descriptive enough?
```suggestion
  FIELDNAM: quarter cycle
```